app/components/welcome/terms-of-use.js

```diff
@@ -0,0 +1,484 @@
+import React from 'react'
```

Comment:
```suggestion
import React from 'react'
import Anchor from '../anchor'
```

For external links to open in browser, not the app.